### ISSUE TITLE
detecting the index.js file for correct output of testing by the main-file.js

### ISSUE THREAD

**Issue by bikenik** [SamVerschueren/alfy-test#2]
Hello! 
This section ([alfy-test/lib/main-file.js](https://github.com/SamVerschueren/alfy-test/blob/0077b1df844414dc484f8f9dbcd240dd26712bc8/lib/main-file.js#L7)) isn't finding my main file (alfred-worflow/index.js) by the provided that I have several `workflow.input.scriptfilter` objects.

If I change `script = object.config.script` to `script = object.config.scriptfile` this resolves this issue.
```js
for (const object of info.objects) {
	if (object.config && object.config.script && object.config.scriptfile !== '') {
		script = object.config.scriptfile;
	}
}
```

Thanks in advance for any response.

**Comment by SamVerschueren** [comment]
@bikenik Why did you close this, might be a valid use case, just din't have much time to check it.

**Comment by bikenik** [comment]
@SamVerschueren you're right, but I thought the time for response will be a measure for the relevance of this issue )

**Comment by SamVerschueren** [comment]
Is your workflow on GitHub? If so, it would come in handy to see it for myself.

**Comment by bikenik** [comment]
Yes, sure. https://github.com/bikenik/alfred-ldoce
